README.md

```diff
@@ -66,19 +67,19 @@ ow.actions.invoke('sample').then(result => console.log(result))
 _Client constructor supports the following mandatory parameters:_
 
 - **apihost.** Hostname and optional port for openwhisk platform, e.g. `openwhisk.ng.bluemix.net` or `my_whisk_host:80`. Used with API URL template `${protocol}://${apihost}/api/v1/`. If port is missing or port value is 443 in the apihost string, protocol is HTTPS. Otherwise, protocol is HTTP.
-- **api_key.** Authorisation key for user account registered with OpenWhisk platform.
+- **apiKey.** Authorisation key for user account registered with OpenWhisk platform.
 
 *Client constructor supports the following optional parameters:*
 
 - **api.** Full API URL for OpenWhisk platform, e.g. `https://openwhisk.ng.bluemix.net/api/v1/`. This value overrides `apihost` if both are present.
 - **namespace**. Namespace for resource requests, defaults to `_`.
-- **ignore_certs**. Turns off server SSL/TLS certificate verification. This allows the client to be used against local deployments of OpenWhisk with a self-signed certificate. Defaults to false.
-- **apigw_token**. API Gateway service authentication token. This is mandatory for using an external API Gateway service, rather than the built-in api gateway.
-- **apigw_space_guid**. API Gateway space identifier. This is optional when using an API gateway service, defaults to the authentication uuid.
+- **ignoreCerts**. Turns off server SSL/TLS certificate verification. This allows the client to be used against local deployments of OpenWhisk with a self-signed certificate. Defaults to false.
+- **apigwToken**. API Gateway service authentication token. This is mandatory for using an external API Gateway service, rather than the built-in api gateway.
+- **apigwSpaceGuid**. API Gateway space identifier. This is optional when using an API gateway service, defaults to the authentication uuid.
 
```

Comment:
I don't think we should be changing these parameters. Moving to another arbitrary style for the constructor parameters does not add any value. I think that "underscores" are more common in JS and it's less error-prone than capitalisation. It's easy to miss a capital letter when you are reading documentation.